### ISSUE TITLE
Crystal element - allow interactive orientation

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#139]
Need to:
- [x] add vectors of the crystal orientation to the drop-down menu of the crystal element;
- [x] allow interactive execution of the `find_orient()` method of the `SRWLOptCryst` class based on the energy provided by user;
- [x] update importer;
- [x] implement code for crystal rotation in JS and bind 2 watches (see `TODO(mrakitin)` in `srw.js`), then get the rotation code back to `sirepo/template/srw.py`;
- [ ] test the code and compare the results with SRW calculations with different input parameters.

**Comment by mrakitin** [comment]
Tested on different combinations of input values, e.g.:
```python
        _v.op_DCM_e0 = 10000.0
        _ang_dif_pl = 2.0
        angAs = 0.3
        _v.op_DCM_ac1 = 0.5
        _v.op_DCM_ac2 = 0.5
        dc = 4.0
        psi[0] = 3.0
        psi[1] = -3.0
```
Result in SRW example (Python):
```python
/home/vagrant/.pyenv/shims/python /home/vagrant/src/radiasoft/sirepo/tests/importer_data/srx.py
Crystal 1: 0.670748313123, 0.108406733307, 0.733719756185, 0.645541104875, 0.401778757206
Crystal 2: 0.254997025572, 0.108406733307, 0.96084571973, -0.895322921962, 0.401778757206
```
Result in Sirepo (in exported file):
```python
    opCr.set_orient(0.670748313123, 0.108406733307, 0.733719756185, 0.645541104875, 0.401778757206)
    opCr.set_orient(0.254997025572, 0.108406733307, 0.96084571973, -0.895322921962, 0.401778757206)
```
Results are identical, so the implementation looks to be correct.

**Comment by mrakitin** [comment]
Implemented in ddbfc64, 41c2923, adfd5b0, cbf07ac and 39847fc. Closing.
